### ISSUE TITLE
Problem in compiling Xamarin.iOS project with Xamarin.Forms

### ISSUE THREAD

**Issue by homerokzam** [joeisapro/MonoTouch.Fabric#8]
I'm trying to use the library in a Xamarin.Forms project, however I'm using it having the following problem when generating for the device (for the emulator it works):

/Users/HomeroKzam/Projects/ProvaDeVida/iOS/MTOUCH: Error MT5214: Native linking failed, undefined symbol: _CLSLogv. This symbol was referenced by the managed member MonoTouch.Fabric.Crashlytics.Crashlytics.CLSLogv. Please verify that all the necessary frameworks have been referenced and native libraries linked. (MT5214) (ProvaDeVida.iOS)

And, do you intend to update the library?

Thanks,
Homero

**Comment by joeisapro** [comment]
I can't replicate the issue you're having, but I also don't use Xamarin.Forms.  Would you be able to send me a repro?

As for updating, I've been wanting to for a long time, but newer versions of Crashlytics (when used with Xamarin) create two log reports for each crash (a Fatal and non-fatal) and I much prefer this version.

If there's enough demand, I can make newer bindings publicly available.

**Comment by homerokzam** [comment]
The project is in: https://github.com/homerokzam/ProvaDeVida

Ok. Thank you very much for your attention.

**Comment by joeisapro** [comment]
Alright, I downloaded your project and was able to successfully build for a device without changing anything.  Try doing a 'Clean All' and rebuild again.

**Comment by homerokzam** [comment]
I cleaned the build and the error continues. I re-created the app, but it does not continue with the same problem.

I'm using Visual Studio for Mac. Are you using Xamarin or VS?

Thanks,
Homero

**Comment by joeisapro** [comment]
I'm still using Xamarin Studio.  I've learned over the last few years to never rush upgrading my Xamarin products, unless I want to lower productivity.

I do have a spare Mac that I can upgrade, however, and I'll let you know how it goes.

PS: Am I to understand this is no longer an issue since you've re-created the app?

**Comment by homerokzam** [comment]
No. I was wrong. The error continues while creating a new project.

I will install Xamarin Studio again and check if this error happens.

Thanks,
Homero

**Comment by joeisapro** [comment]
Ok, I upgraded my toolchain using the stable channel.
Everything still worked fine.  Then, I upgraded to VS for Mac and started getting the same linker errors.

I loaded Xamarin Studio, reverted back to stable and the errors are gone.

I will get to the bottom of this, eventually, but don't know when that will happen.

**Comment by joeisapro** [comment]
This is my system info, after reverting to stable from XS:

Xamarin Studio Enterprise
Version 6.1.2 (build 44)
Installation UUID: dfbfd473-0275-4e49-84a4-94e520fced66
Runtime:
	Mono 4.6.2 (mono-4.6.0-branch/08fd525) (64-bit)
	GTK+ 2.24.23 (Raleigh theme)

	Package version: 406020007

NuGet
Version: 3.4.3.0

Xamarin.Profiler
Not Installed

Xamarin.Android
Version: 7.0.2.37 (Visual Studio Enterprise)
Android SDK: /Users/joepro/Library/Developer/Xamarin/android-sdk-macosx
	Supported Android versions:
		4.0.3 (API level 15)
		6.0   (API level 23)
		7.0   (API level 24)

SDK Tools Version: 25.1.2
SDK Platform Tools Version: 24.0.0
SDK Build Tools Version: 23.0.2

Java SDK: /usr
java version "1.8.0_101"
Java(TM) SE Runtime Environment (build 1.8.0_101-b13)
Java HotSpot(TM) 64-Bit Server VM (build 25.101-b13, mixed mode)

Android Designer EPL code available here:
https://github.com/xamarin/AndroidDesigner.EPL

Xamarin Android Player
Not Installed

Apple Developer Tools
Xcode 8.2 (11766)
Build 8C38

Xamarin.Mac
Version: 2.10.0.113 (Visual Studio Enterprise)

Xamarin.iOS
Version: 10.3.0.0 (Visual Studio Enterprise)
Hash: e02d272
Branch: xcode8.2
Build date: 2016-12-12 23:00:41-0500

Xamarin Inspector
Version: 1.0.0.0
Hash: 1f3067d
Branch: master
Build date: 2016-11-15 4:13:59 PM

Build Information
Release ID: 601020044
Git revision: 0ccfcd52b95305ebd5b7eca0d88c1017035910ae
Build date: 2016-10-28 15:12:43-04
Xamarin addins: a39a869d8a78d87bdc6775f696c13a4cc9024501
Build lane: monodevelop-lion-cycle8

Operating System
Mac OS X 10.12.0
Darwin Joepros-iMac.local 16.0.0 Darwin Kernel Version 16.0.0
    Mon Aug 29 17:56:20 PDT 2016
    root:xnu-3789.1.32~3/RELEASE_X86_64 x86_64

Enabled user installed addins
Xamarin Inspector 1.0.0.0
